### DISCUSSION TITLE
May I ask if the current version of thorium with AVX2 is purely compiled with AVX2 or is it only the kernel that is?

### DISCUSSION THREAD

**Discussion by Malus-risus** [Alex313031/thorium#562] [General]
May I ask if the current version of thorium with AVX2 is purely compiled with AVX2 or is it only the kernel that is?

**Comment by gz83** [comment]
I'm not sure what you want to express. If you want to know the compilation parameters of the current AVX2 version, you can check it in the 'other' folder in this repo. At the same time, you also need to know about LLVM and SIMD.

**Comment by Alex313031** [comment]
@Malus-risus What do you mean "kernel". A kernel is an operating system component.

This is a fork of Chromium. Everything is compiled with AVX2 in the AVX2 version.
